java/src/test/java/ai/rapids/cudf/TableTest.java

```diff
@@ -2392,6 +2392,65 @@ void testWindowingRowNumber() {
     }
   }
 
+  @Test
+  void testWindowingCollect() {
+    Aggregation aggCollect = Aggregation.collect();
+    WindowOptions winOpts = WindowOptions.builder()
+                                         .minPeriods(1)
+                                         .window(2, 1).build();
+    StructType nestedType = new StructType(false,
+        new BasicType(false, DType.INT32), new BasicType(false, DType.STRING));
+    try (Table raw = new Table.TestBuilder()
+             .column( 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1) // GBY Key
+             .column( 1, 1, 1, 1, 2, 2, 2, 2, 3, 3, 3, 3) // GBY Key
+             .column( 1, 1, 2, 2, 3, 3, 4, 4, 5, 5, 6, 6) // OBY Key
+             .column( 7, 5, 1, 9, 7, 9, 8, 2, 8, 0, 6, 6) // Agg Column of INT32
+             .column(nestedType, /* Agg Column of Struct */
+                 new StructData(1, "s1"), new StructData(2, "s2"), new StructData(3, "s3"),
+                 new StructData(4, "s4"), new StructData(11, "s11"), new StructData(22, "s22"),
+                 new StructData(33, "s33"), new StructData(44, "s44"), new StructData(111, "s111"),
+                 new StructData(222, "s222"), new StructData(333, "s333"), new StructData(444, "s444")
+              ).build();
+         ColumnVector expectSortedAggColumn = ColumnVector.fromBoxedInts(7, 5, 1, 9, 7, 9, 8, 2, 8, 0, 6, 6)) {
+      try (Table sorted = raw.orderBy(Table.asc(0), Table.asc(1), Table.asc(2))) {
```

Comment:
The code formatting seems wrong. Line#2415 should be nested inside the previous block.

Reply:
Seems it is already inside the previous block as below
 ```
try(Table raw = .....;
      ColumnVector expectSortedAggColumn = ....) {
    try (Table sorted = raw.orderBy(Table.asc(0), Table.asc(1), Table.asc(2))) { 
        ....
    }
}
```

Reply:
My mistake. This seems to be alright.